### PR TITLE
Cosmos SDK 4 code review / cleanup

### PR DESCRIPTION
**Pull request by bartelink**
re #197 #203 Random changes as I review the code

### CODE REVIEW

File: src/Equinox.Cosmos/Cosmos.fs [hunk 1]
```diff
@@ -1179,7 +1165,7 @@ type Discovery =
             UriAndKey (Uri uri, key)
         | _ -> invalidArg "connectionString" "unrecognized connection string format; must be `AccountEndpoint=https://...;AccountKey=...=;`"
 
-type EquinoxCosmosOperationsFactory
```
Comment (ylibrach) [inline src/Equinox.Cosmos/Cosmos.fs]: You really hate the `Equinox` in front, huh? ;)

Comment (bartelink) [reply src/Equinox.Cosmos/Cosmos.fs]: Maybe ;) as mentioned, there;'s nothing with `Equinox` in the type name anywhere else so the removal is driven by an ill-defined consistency instinct....

Comment (bartelink) [reply src/Equinox.Cosmos/Cosmos.fs]: it gets worse :( 

I've pended renaming it to `StoreOperationsFactory.Create() : Equinox.Cosmos.Store.Operations`

Am going to spend some quality time with my thesaurus (and PoeSA book) as any glee you might imagine me having is not here...

Comment (bartelink) [reply src/Equinox.Cosmos/Cosmos.fs]: I'm coming back to thinking it should be called [something including] `Gateway` based on https://martinfowler.com/eaaCatalog/gateway.html

`Equinox.Cosmos.Store` currently has short names and its not a mainstream direct usage thing

if you don't hate the name, then I'd try to find a better name for the one that's in `.Internal` atm ?

**Comment by bartelink** [comment]
Merging this - I'm currently doing a follow-on cleanup exercise, but want to release the alpha from `cosmossdk4` so @ylibrach can test the integration

**Pull request merged**